src/StockportWebapp/Controllers/AtoZController.cs

```diff
@@ -39,6 +41,9 @@ public async Task<IActionResult> Index(string letter)
             Breadcrumbs = new List<Crumb>()
         };
 
+        if(await _featureManager.IsEnabledAsync("SiteRedesign"))
```

Comment:
It's the most picky thing in the world (sorry) but could you pop a space between if and ( please